<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmProgressBarPercent</class>
 <widget class="QWidget" name="frmProgressBarPercent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="ProgressBarPercent" name="widget6" native="true"/>
   </item>
   <item row="1" column="1">
    <widget class="ProgressBarPercent" name="widget5" native="true"/>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QSlider" name="horizontalSlider">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="ProgressBarPercent" name="widget4" native="true"/>
   </item>
   <item row="0" column="2">
    <widget class="ProgressBarPercent" name="widget3" native="true"/>
   </item>
   <item row="0" column="1">
    <widget class="ProgressBarPercent" name="widget2" native="true"/>
   </item>
   <item row="0" column="0">
    <widget class="ProgressBarPercent" name="widget1" native="true"/>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QWidget" name="widget_3" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>90</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>180</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>270</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QRadioButton" name="rbtnClockWise">
           <property name="text">
            <string>顺时针</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbtn">
           <property name="text">
            <string>逆时针</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="rbtnArc">
           <property name="text">
            <string>圆弧</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbtnPolo">
           <property name="text">
            <string>水池</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbtnArcPolo">
           <property name="text">
            <string>圆弧水池</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbtnWave">
           <property name="text">
            <string>水纹</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ProgressBarPercent</class>
   <extends>QWidget</extends>
   <header>progressbarpercent.h</header>
   <container>1</container>
   <slots>
    <slot>setValue(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>41</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
